explainability-service/src/main/java/org/kie/trustyai/service/data/exceptions/StorageWriteException.java

```diff
@@ -0,0 +1,7 @@
+package org.kie.trustyai.service.data.exceptions;
+
+public class StorageWriteException extends Exception {
```

Comment:
Let's use unchecked exceptions :)
```suggestion
public class StorageWriteException extends RuntimeException {
```